src/Mention.tsx

```diff
@@ -7,9 +7,33 @@ interface MentionProps {
 
     /** The color of the highlight. */
     color?: string;
+
+    /** Whether to use text color highlighting instead of background color. */
+    highlightTextColor?: boolean;
 }
 
-const Mention: React.FC<MentionProps> = ({ display, color }) => {
+const Mention: React.FC<MentionProps> = ({ display, color, highlightTextColor }) => {
+    if (highlightTextColor) {
+        return (
+            <Box component='span' sx={{ position: 'relative' }}>
+                {display}
+                <Box
+                    component='span'
+                    sx={{
+                        position: 'absolute',
+                        left: '0px',
+                        top: '-2px',
+                        right: '0px',
+                        bottom: '0px',
+                        color: (theme) => theme.palette.primary.main,
```

Comment:
This should probably be `(theme) => getColor(theme.palette.mode, color)`. Then the highlighted text can have the color specified in the same way as currently changing the color of the highlight background

Reply:
oh nice, yeah good addition, this adds flexibility without more code. pushing the update